addons/website/tests/test_page.py

```diff
@@ -212,14 +212,15 @@ def test_unpublished_page(self):
                     </t>''',
             'key': 'test.base_view',
         })
-        generic_page = Page.create({
```

Comment:
is there any reason to rename it ? Generic is not to say "the object page", but a page shared between several websites.
It allow to easily understand in the code that the page has not yet been forked and is not specific to a website.


the real test should be:
`specific_page = generic_page.with_context({'website_id': self.env['website'].get_current_website().id}).write({'is_published': False, 'arch': generic_page.arch.replace('I am a ...`
But the problem is just that we need to redo a search to find the new forked view (since write return a bool), so copy is simple way since we want just check that the state active from forked view take the priority in the generic.

Reply:
> is there any reason to rename it ? Generic is not to say "the object page", but a page shared between several websites.

Not fundamentally, just seemed that the page would implicitly be generic since it's part of the testcase and not any specfic case. I see your point.